pkgs/development/tools/sq/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "sq";
+  version = "0.15.4";
+
+  src = fetchFromGitHub {
+    owner = "neilotoole";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-4CINLOHUVXQ+4e5I1fMqog6LubMm8RnbFmeuBOwALaw=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  vendorSha256 = "sha256-lNpWXKtnzwySzinNPxAKuaLqweWuS6zz8s2W4xXWlqM=";
+
+  # Some tests violates sandbox constraints.
+  doCheck = false;
+
+  postInstall = ''
+    installShellCompletion --cmd sq \
+      --bash <($out/bin/sq completion bash) \
+      --fish <($out/bin/sq completion fish) \
+      --zsh <($out/bin/sq completion zsh)
+  '';
+
+
+  meta = with lib; {
+    description = "swiss-army knife for data";
```

```suggestion
    description = "Swiss army knife for data";
```
[First letter should be capitalised.](https://nixos.org/manual/nixpkgs/stable/#var-meta-description) Also I have never seen it with a dash. Translators don't write it with a dash either.